src/Bridge/Doctrine/Orm/Extension/EagerLoadingExtension.php

```diff
@@ -88,9 +87,13 @@ private function joinRelations(QueryBuilder $queryBuilder, string $resourceClass
             $select = [];
             $targetClassMetadata = $queryBuilder->getEntityManager()->getClassMetadata($mapping['targetEntity']);
 
-            //@TODO useless at the moment, but Doctrine queries non-necessary relations because addSelect($associationAlias) lazy fetch every association
-            //By using this array of properties, it Lazy fetch association, which are initially in an INNER JOIN (fetches them twice ?!)
-            //If I try to use associationAlias when it has every property, but the properties array when it doesn't it fails because it can not fetch parent association
+            //@TODO useless at the moment, but Doctrine queries non-necessary relations because addSelect($associationAlias) will try to fetch every
+            //association belonging to the association, but if those are not serialized we don't need them ! Can be especially anoying when
+            //the relation is a reverse relation (so doctrine does lots of subqueries). Here I need a way to tell doctrine to take only what we need.
+            //
+            //By using this array of properties, Doctrine will lazy fetch every associations, which are initially in an INNER JOIN (fetches them twice ?!)
+            //If I try to use associationAlias when it has every property (so that Doctrine does not do sub queries on top of the initial one) it fails because it can not fetch parent association (like internally they try to find aliasX, but no aliasX is selected, we've instead aliasX.id, aliasX.name)
+            //I obviously tried to specify `WITH` `aliasX.id = aliasY.id` without success
             foreach ($this->getMetadataProperties($mapping['targetEntity']) as $property => $propertyMetadata) {
```

Comment:
Please read my story, if you have any clue or idea to help me think it'd be wonderful :D. ping @Simperfit